packages/@cdktf/functions/scripts/generate.ts

```diff
@@ -0,0 +1,31 @@
+// uses the functions.json file to generate the bindings for CDKTF
+
+import fs from "fs/promises";
+import * as path from "path";
+import generate from "@babel/generator";
+import template from "@babel/template";
+import * as t from "@babel/types";
+import prettier from "prettier";
+import { FUNCTIONS_METADATA_FILE } from "./fetch-metadata";
+
+async function fetchMetadata() {
+  const file = path.join(__dirname, FUNCTIONS_METADATA_FILE);
```

Comment:
Mainly speed of development when iterating on the code generation 😄 Especially since this required a beta version of Terraform (see the just updated README of the tool).

A future thought (that is not yet worth deciding architecture decisions on) could be that we'd have multiple of these files and generate function bindings for each supported Terraform version.